core/src/main/java/org/radargun/stats/BasicStatistics.java

```diff
@@ -30,14 +33,33 @@ public BasicStatistics(OperationStats prototype) {
    }
 
    protected OperationStats createOperationStats(int operationId) {
-      return prototype.newInstance();
+      return prototype.copy();
    }
 
    public Statistics newInstance() {
       return new BasicStatistics(prototype);
    }
 
    @Override
+   public void registerOperationsGroup(String name, Set<Operation> operations) {
+      if (groupOperationsMap.containsKey(name)) {
+         return;
```

Comment:
The original PR was throwing an exception for this case, any reason you changed that?